carousel.html

```diff
@@ -35,10 +37,29 @@
   </header>
 
     <main>
+      <div class="carousel" id="carousel">
+        
+        <button class="carousel__button carousel__button--back" id="back-button">
+          <svg xmlns="http://www.w3.org/2000/svg" height="32" viewBox="0 96 960 960" width="32"><path xmlns="http://www.w3.org/2000/svg" fill="#fff" d="M561 816 320 575l241-241 43 43-198 198 198 198-43 43Z"/></svg>
```

Comment:
Only because during reloading the page those icons were stretching (I suppose that was because they were uploaded from the google materials in their default size and it took some time to style them with my css, right?).